source/_integrations/time_date.markdown

```diff
@@ -9,9 +9,11 @@ ha_release: pre 0.7
 ha_qa_scale: internal
 ---
 
-The time and date (`time_date`) sensor platform simple displays the time in various formats, the date, or both.
+The time and date (`time_date`) sensor platform adds one or more sensors to your Home Assistant state machine.
+It also displays the time in various formats, the date, or both in non-Lovelace mode.  
```

Comment:
Let's remove the reference of the non-Lovelace mode.

Reply:
That's the main point here. This integration was used to display time/tima and date in pre-Lovelace times.
And to be able to do so it creates some sensors.
Now you can see time displayed only in non-Lovelace, there is no time displayed in Lovelace afaik.
But the sensors are still in state machine.

Reply:
There is no difference. You can still add the to Lovelace.
States pages were generated, Lovelace is a manual job (unless running in automated mode). You could add this note you made there, to every single integration and entity throughout the documentation, hence me suggesting to remove it.

Reply:
Ah, I understand.
If it's pointless to have such a note in every single integration, where can we have it?
I believe it's worth to  convey this concept to the users.

Reply:
It is the basic core concept of Lovelace?

Reply:
You mean that in Satest mode everything is added automatically and one need to hide unnecessary stuff when in Lovelace only the manually added stuff is displayed?
Yes, that's correct.
Perhaps the main reason of this PR was the way to define sensors - I assumed that everything is in the state machine, but only sensors listed in `display_options` will be displayed and it's only applicable to the States mode.